src/store/indexeddb-local-backend.js

```diff
@@ -239,42 +223,39 @@ LocalIndexedDBStoreBackend.prototype = {
     },
 
     /**
-     * Persist a list of User objects. Users with the same 'userId' will be
-     * replaced.
-     * @param {User[]} users An array of users
+     * Persist a list of [user id, presence event] they are for.
+     * Users with the same 'userId' will be replaced.
+     * Prfesence events should be the event in its raw form (not the Event
+     * object)
+     * @param {Object[]} tuples An array of [userid, event] tuples
```

Comment:
Why tuples, which you need to consistently remember which index has what, over an `Object` where the key names are self-documenting?